Formatting stylization

phg49389 opened the pull request:
Went through the code base and cleaned it up. Organized imports, changed a lot of quote marks to apostrophes, did a few small bug fixes, and increased the display size of the Description field being rendered by the templates.

e-jameson commented on app/forms.py:
```diff
@@ -35,43 +40,43 @@ def _safe(form, field):
 class RegularPostForm(RenderableForm):
-    id = HiddenField("", [validators.DataRequired(), validators.Length(max=8)])
-    submitters_username = HiddenField("", [validators.DataRequired(), validators.Length(max=50)])
+    id = HiddenField('', [validators.DataRequired(), validators.Length(max=8)])
```
Can you use a different name that doesn't collide with the built in keyword id? PEP8 would say to use `id_` or a different name altogether.

e-jameson commented on app/forms.py:
```diff
@@ -35,43 +40,43 @@ def _safe(form, field):
 class ExternalPosterForm(RenderableForm):
-    id = HiddenField("", [validators.DataRequired(), validators.Length(max=8)])
-    submitters_username = StringField("Email of external poster:", [validators.DataRequired(),
+    id = HiddenField('', [validators.DataRequired(), validators.Length(max=8)])
```
Same here

e-jameson commented on app/models.py:
```diff
@@ -50,7 +47,7 @@ def __init__(self, username, first, last, email, phone):
 
     def __repr__(self):
-        return "<Contact '%s'>" % self.first_name + " " + self.last_name
+        return "<Contact '%s'>" % self.first_name + ' ' + self.last_name
```
its kinda weird using both `%s` notation and `+ ' ' +`. Could you move the name combination to the line above and use `%s` for both?

phg49389 commented:
I've gone through and done a lot of code refactoring; organized the imports, replaced most of the quotes with apostrophes, removed some old comments and print statements, redid some logic, redid a few naming conventions, and added an __all__ dunder to controller.py to simplify the imports, fixed a few display errors, and added Josiah's new version of the logout method.
